src/Kestrel/Kestrel.csproj

```diff
@@ -17,9 +17,6 @@
   <ItemGroup>
     <ProjectReference Include="..\Kestrel.Core\Kestrel.Core.csproj" />
     <ProjectReference Include="..\Kestrel.Https\Kestrel.Https.csproj" />
-
-    <!-- Even though the Libuv transport is no longer used by default, it remains for back-compat -->
```

Comment:
@natemcmaster I left this comment the first time I switched the default transport to managed sockets. Doesn't removing this dependency break back-compat?